### PR TITLE
Move integrations to System Health

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

CleanShot 2022-04-28 at 20 27 54

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by matthiasdebaat** [comment]
Nice!

File: src/panels/config/info/integrations-card.ts [hunk 1]
```diff
@@ -151,54 +107,25 @@ class IntegrationsCard extends LitElement {
   }
 
   private async _fetchSetups() {
-    const setups = {};
-    for (const setup of await fetchIntegrationSetups(this.hass)) {
-      setups[setup.domain] = setup;
-    }
-    this._setups = setups;
+    const setups = await fetchIntegrationSetups(this.hass);
+    this._setups = setups.filter((setup) => setup.seconds && setup.seconds > 1);
```
Comment (bramkragten) [inline src/panels/config/info/integrations-card.ts:111]: Can we order it on setup time?

File: src/panels/config/info/integrations-card.ts [hunk 2]
```diff
@@ -151,54 +107,27 @@ class IntegrationsCard extends LitElement {
...
 
   private async _fetchSetups() {
-    const setups = {};
-    for (const setup of await fetchIntegrationSetups(this.hass)) {
-      setups[setup.domain] = setup;
-    }
-    this._setups = setups;
+    const setups = await fetchIntegrationSetups(this.hass);
+    this._setups = setups
+      .filter((setup) => setup.seconds && setup.seconds > 1)
```
Comment (balloob) [inline src/panels/config/info/integrations-card.ts:112]: No need to filter if we sort by seconds. The slowest are on the bottom.

**Review by bramkragten — DISMISSED** [review]

File: src/panels/config/info/integrations-card.ts (hunk 2 above)
Comment (balloob) [reply src/panels/config/info/integrations-card.ts:112]: sorry that's what I meant

**Review by bramkragten — APPROVED** [review]

**Pull request merged**